cime_config/acme/allactive/config_pes.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 
-<config_pes>
+<config_pes version="2.0">
```

Comment:
This needs to be added to acme/allactive/config_pesall.xml as well.  That's the file we're currently using.